client/app/dashboard/model/list/ModelList.js

```diff
@@ -713,6 +713,8 @@ angular.module('dashboard.Dashboard.Model.List', [
   
   $scope.$watch('pagingOptions', function (newVal, oldVal) {
     if (newVal.currentPage != oldVal.currentPage || newVal.pageSize != oldVal.pageSize) {
+      var currentPageCount = Math.ceil($scope.totalServerItems / newVal.pageSize);
+      $('.ngPagerCurrent:input[type="number"]').attr({'max': currentPageCount});
```

Comment:
Is it possible to do this in a more Angular way?

Put it on $scope.currentPageCount, and then in the Jade/HTML do
<input max="{{ currentPageCount }}"> ?

Reply:
HI @tma-isbx , i think the input was generated by the ui-grid itself, thats why i cant do it this way i think.

Reply:
Oops, yeah, I see.